packages/tracing/src/browser/metrics.ts

```diff
@@ -17,7 +17,7 @@ const global = getGlobalObject<Window>();
 
 /** Class tracking metrics  */
 export class MetricsInstrumentation {
-  private _measurements: Measurements = {};
+  private _measurements: Record<string, { value: number }> = {};
```

Comment:
do we need to add units to any of these? Things like LCP are in seconds.

Reply:
Added units here: afcdaeeb3876c5030fea635814e13a13c7451a94